testsupport/wait/host.go

```diff
@@ -770,24 +769,27 @@ func (a *HostAwaitility) WaitAndVerifyThatUserSignupIsNotCreated(t *testing.T, n
 }
 
 // WaitForBannedUser waits until there is a BannedUser available with the given email
-func (a *HostAwaitility) WaitForBannedUser(t *testing.T, email string) (*toolchainv1alpha1.BannedUser, error) {
-	t.Logf("waiting for BannedUser for user '%s' in namespace '%s'", email, a.Namespace)
+// !!! WARNING: for now, just used for WA
+func (a *HostAwaitility) WaitForBannedUser(t *testing.T, userEmailHash string) (*toolchainv1alpha1.BannedUser, error) {
+	t.Logf("waiting for BannedUser for user '%s' in namespace '%s'", userEmailHash, a.Namespace)
```

Comment:
probably you'll want to print the decoded email here , but not sure.

Reply:
There is no way, I think. User email hash is encoded in MD5: https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/hash/encode.go